public/js/tracking_protection.js

```diff
@@ -44,5 +54,18 @@ if (!_dntEnabled()) {
 if(typeof(ga) !== "undefined") {
 	ga("create", "UA-77033033-16");
 	ga("set", "anonymizeIp", true);
-	ga("send", "pageview");
+
+
+  // Strip url params if they contain token or hash
+  const loc = document.location;
+  let pageValue = loc.pathname + loc.search;
+  if (document.location.search &&
+      (document.location.search.indexOf("token") !== -1 ||
+       document.location.search.indexOf("hash") !== -1)
```

Comment:
Same here, we could possibly use [`String#includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) which returns a Boolean (and use our handy `loc` variable from L60):

```js
if (loc.search && (loc.search.includes("token") || loc.search.includes("hash"))) {..}
```